core/roboconf-core/src/main/java/net/roboconf/core/model/RuntimeModelValidator.java

```diff
@@ -25,6 +25,50 @@
+import static net.roboconf.core.errors.ErrorCode.RM_CYCLE_IN_COMPONENTS_INHERITANCE;
+import static net.roboconf.core.errors.ErrorCode.RM_CYCLE_IN_FACETS_INHERITANCE;
+import static net.roboconf.core.errors.ErrorCode.RM_DOT_IS_NOT_ALLOWED;
+import static net.roboconf.core.errors.ErrorCode.RM_EMPTY_COMPONENT_INSTALLER;
+import static net.roboconf.core.errors.ErrorCode.RM_EMPTY_COMPONENT_NAME;
+import static net.roboconf.core.errors.ErrorCode.RM_EMPTY_FACET_NAME;
+import static net.roboconf.core.errors.ErrorCode.RM_EMPTY_INSTANCE_COMPONENT;
+import static net.roboconf.core.errors.ErrorCode.RM_EMPTY_INSTANCE_NAME;
+import static net.roboconf.core.errors.ErrorCode.RM_EMPTY_VARIABLE_NAME;
+import static net.roboconf.core.errors.ErrorCode.RM_INVALID_APPLICATION_EXPORT_PREFIX;
+import static net.roboconf.core.errors.ErrorCode.RM_INVALID_APPLICATION_NAME;
+import static net.roboconf.core.errors.ErrorCode.RM_INVALID_APPLICATION_VERSION;
+import static net.roboconf.core.errors.ErrorCode.RM_INVALID_COMPONENT_INSTALLER;
+import static net.roboconf.core.errors.ErrorCode.RM_INVALID_COMPONENT_NAME;
+import static net.roboconf.core.errors.ErrorCode.RM_INVALID_EXTERNAL_EXPORT;
+import static net.roboconf.core.errors.ErrorCode.RM_INVALID_FACET_NAME;
+import static net.roboconf.core.errors.ErrorCode.RM_INVALID_INSTANCE_NAME;
+import static net.roboconf.core.errors.ErrorCode.RM_INVALID_INSTANCE_PARENT;
+import static net.roboconf.core.errors.ErrorCode.RM_INVALID_RANDOM_KIND;
+import static net.roboconf.core.errors.ErrorCode.RM_INVALID_VARIABLE_NAME;
+import static net.roboconf.core.errors.ErrorCode.RM_MAGIC_INSTANCE_VARIABLE;
+import static net.roboconf.core.errors.ErrorCode.RM_MISSING_APPLICATION_DSL_ID;
+import static net.roboconf.core.errors.ErrorCode.RM_MISSING_APPLICATION_EXPORT_PREFIX;
+import static net.roboconf.core.errors.ErrorCode.RM_MISSING_APPLICATION_GEP;
+import static net.roboconf.core.errors.ErrorCode.RM_MISSING_APPLICATION_GRAPHS;
+import static net.roboconf.core.errors.ErrorCode.RM_MISSING_APPLICATION_NAME;
+import static net.roboconf.core.errors.ErrorCode.RM_MISSING_APPLICATION_VERSION;
+import static net.roboconf.core.errors.ErrorCode.RM_MISSING_INSTANCE_PARENT;
+import static net.roboconf.core.errors.ErrorCode.RM_MISSING_VARIABLE_VALUE;
+import static net.roboconf.core.errors.ErrorCode.RM_NOT_A_ROOT_COMPONENT;
+import static net.roboconf.core.errors.ErrorCode.RM_NO_ROOT_COMPONENT;
+import static net.roboconf.core.errors.ErrorCode.RM_NO_VALUE_FOR_RANDOM;
+import static net.roboconf.core.errors.ErrorCode.RM_ORPHAN_FACET;
+import static net.roboconf.core.errors.ErrorCode.RM_ORPHAN_FACET_WITH_CHILDREN;
+import static net.roboconf.core.errors.ErrorCode.RM_ROOT_INSTALLER_MUST_BE_TARGET;
+import static net.roboconf.core.errors.ErrorCode.RM_UNREACHABLE_COMPONENT;
+import static net.roboconf.core.errors.ErrorCode.RM_UNRESOLVABLE_FACET_VARIABLE;
+import static net.roboconf.core.errors.ErrorCode.RM_UNRESOLVABLE_VARIABLE;
```

Comment:
Indeed, but this is a bad habit, as we might import things we do not want.
This is why we have a Checkstyle rule to verify we do not use wildcard imports.